README.adoc

```diff
@@ -75,45 +75,37 @@ Windows, see link:README.win32.adoc[].
 The OCaml manual is distributed in HTML, PDF, Postscript, DVI, and Emacs
 Info files.  It is available at
 
-http://caml.inria.fr/
-
-The community also maintains the Web site http://ocaml.org, with tutorials
-and other useful information for OCaml users.
+http://ocaml.org/
```

Comment:
Can we give a more precise link to where the OCaml manual is? For caml.inria.fr, that would be <http://caml.inria.fr/pub/docs/manual-ocaml/> (which points to versions in other formats as well). What is the corresponding ocaml.org link? 

(<http://caml.inria.fr/pub/distrib/ocaml-4.05/> would also be a natural choice of URL but it has the downside of being specific to an OCaml version, and we don't want to have to change the URL on each release; if we kept pointing at caml.inria.fr it would make sense to add a distrib/ocaml-latest directory. We should be careful of having the same release-robustness property on ocaml.org.)

Reply:
I didn't add more detail because the previous link was also to the old homepage, so it would take a user a similar number of clicks to reach the manual. If you feel it would help though, I can point directly to the manual, which of course remains at http://caml.inria.fr/pub/docs/manual-ocaml/

Reply:
Yes, please point to the manual directly (on caml.inria.fr or ocaml.org).